docs/rfcs/016-holdings_service/README.md

```diff
@@ -0,0 +1,128 @@
+# **RFC 016: Holdings service**
+
+**Last updated: 23 September 2019.**
+
+## **Background**
+
+As we move a greater proportion of people over from wellcomelibrary.org to
```

Comment:
Can we headline the requirement?

"In order to provide a modern ordering service to users we need to replace the existing library holdings service."

Reply:
Might leave modern out, sound archaic.